### PR TITLE
add some code for kyoku (WIP)

### PR DESCRIPTION
**Pull request by rasca0027**
need some clarification!!!

### CODE REVIEW

File: mahjong/kyoku.py [hunk 1]
```diff
@@ -226,6 +226,9 @@ def bakaze(self, value: Jihai) -> None:
                 "Bakaze should be 1 in Tonpuusen, should be 1 or 2 in Hanchan")
         self._bakaze = value
 
+    def get_oya_player(self):
+        return list(filter(lambda p: p.jilaze == Jihai.TON, self.players))[0]
```
Comment (rasca0027) [inline mahjong/kyoku.py:230]: jikaze

Comment (rasca0027) [reply mahjong/kyoku.py:230]: 改成next

File: mahjong/kyoku.py [hunk 2]
```diff
@@ -316,5 +341,39 @@ def apply_points(self, han: int, fu: int, tsumo: bool, loser: Optional[Player] =
 
 class Game:
     def __init__(self):
+        self.bakaze = Jihai.TON
+        self.kyoku_num = 1  # e.g.東1局
+        self.players = self.get_init_players()
+        self.current_kyoku = Kyoku(players, 0, self.bakaze, 0)
```
Comment (hoaaoh) [inline mahjong/kyoku.py:347]: self.players?

File: mahjong/kyoku.py [hunk 3]
```diff
@@ -316,5 +341,39 @@ def apply_points(self, han: int, fu: int, tsumo: bool, loser: Optional[Player] =
...
+        pass
+
+    def start_game():
+        while True:
+            renchan, kyotaku, honba = self.current_kyoku.start()
+            if check_tobu():  # 有人被飛
+                break
+            if renchan:
+                # oya same player, don't increase kyoku_num or bakaze, increase honba
+                honba += 1
```
Comment (rasca0027) [inline mahjong/kyoku.py:360]: honba已經+1了

File: mahjong/kyoku.py (hunk 2 above)
Comment (rasca0027) [reply mahjong/kyoku.py:347]: 喔。

File: mahjong/kyoku.py [hunk 4]
```diff
@@ -316,5 +341,39 @@ def apply_points(self, han: int, fu: int, tsumo: bool, loser: Optional[Player] =
...
+            else:
+                if self.kyoku_num == 4:
+                    if self.bakaze == Jihai.NAN:
+                        break # end game
+                    elif self.bakaze == Jihai.TON:
+                        self.bakaze = Jihai.NAN
+                        self.kyoku_num = 1
+                else:
+                    self.kyoku_num += 1
+                # TODO: advance player jikaze
```
Comment (rasca0027) [inline mahjong/kyoku.py:370]: https://github.com/rasca0027/Mahjong4RL/blob/master/tests/test_player.py#L57

**Pull request merged**